products/cloudrun/api.yaml

```diff
@@ -0,0 +1,565 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: CloudRun
+display_name: Google Cloud Run
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://run.googleapis.com/v1alpha1/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+objects:
+# Cloud Run DomainMappings
+- !ruby/object:Api::Resource
+  name: DomainMapping
+  kind: DomainMapping
+  base_url: projects/{{project}}/locations/{{location}}/domainmappings
+  references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/run/docs/mapping-custom-domains'
+      api: 'https://cloud.google.com/run/docs/reference/rest/v1alpha1/projects.locations.domainmappings'
+  description: |-
+    Resource to hold the state and status of a user's domain mapping.
+
+    **Note:** Cloud Run as a product is in beta, however the REST API is currently still an alpha.
+    Please use this with caution as it may change when the resource moves to beta.
```

Comment:
suggestion: s/resource/API (since we're in the beta provider right now, which could make things confusing)